src/lightning-address.ts

```diff
@@ -71,14 +71,22 @@ export default class LightningAddress {
     return `https://${this.domain}/.well-known/keysend/${this.username}`;
   }
 
-  async generateInvoice(url: URL): Promise<Invoice> {
```

Comment:
the callback URL here should be used from `this`

Reply:
Done!